pkg/controllers/networkpolicy/policy_combiner.go

```diff
@@ -0,0 +1,61 @@
+package networkpolicy
+
+import (
+	log "github.com/sirupsen/logrus"
+	networkingv1 "k8s.io/api/networking/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sort"
+)
+
+func combinePolicies(nps []*networkingv1.NetworkPolicy) *networkingv1.NetworkPolicy {
+	// policies should be combined in alphabetical order
```

Comment:
Why? It may be worth documenting the reason here.

Reply:
Because they are combined in this way by calico-node, when two policies have the same order then they are sorted alphabetically

Reply:
I'll comment